greenery/fsm.py

```diff
@@ -705,9 +708,7 @@ def derive(self, input: Iterable[alpha_type], /) -> Fsm:
             state = self.initial
             for symbol in input:
                 if symbol not in self.alphabet:
-                    if ANYTHING_ELSE not in self.alphabet:
-                        raise KeyError(symbol)
-                    symbol = ANYTHING_ELSE
+                    raise KeyError(symbol)
```

Comment:
I removed this more as a site note when checking where ``ANYTHING_ELSE`` transformations were used: I am basically sure this is was a bug in the library. This conversion to ``ANYTHING_ELSE`` was never tested and would return something like `parse('.').derive('a') == parse('')` which doesn't seem correct.